org.nuspell.Nuspell.json

```diff
@@ -0,0 +1,64 @@
+{
+  "app-id": "org.nuspell.Nuspell",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "nuspell",
+  "modules" : [
+    {
+      "name": "boost",
+      "buildsystem": "simple",
+      "build-commands": [
+        "./bootstrap.sh --prefix=/app --with-libraries=locale",
+        "./b2 install"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://dl.bintray.com/boostorg/release/1.65.1/source/boost_1_65_1.tar.bz2",
+          "sha256": "9807a5d16566c57fd74fb522764e0b134a8bbe6b6e8967b83afefd30dcd3be81"
+        }
+      ]
+    },
+    {
+      "name": "launcher",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/share/applications/",
+        "cp -f org.nuspell.Nuspell.desktop /app/share/applications/",
+        "mkdir -p /app/share/icons/hicolor/512x512/apps/",
+        "cp -f logo-white-512x512.png /app/share/icons/hicolor/512x512/apps/org.nuspell.Nuspell.png"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "path": "org.nuspell.Nuspell.desktop"
+        },
+        {
+          "type": "file",
+          "url": "https://raw.githubusercontent.com/nuspell/nuspell.github.io/master/assets/images/logo-white-512x512.png",
```

Comment:
Ideally it should be maintained in Nuspell code repository and installed by its build system.